.circleci/config.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     - run:
         command: ./gradlew testcontainers:build -x test -x check
     - run:
-        command: ./gradlew testcontainers:check --scan
+        command: ./gradlew testcontainers:check javadoc --scan
```

Comment:
perhaps we should add the same to AZP's config? Or maybe we can simply make `tasks.check.dependsOn(javadoc)`?

Reply:
Good idea - done.